includes/classes/ExternalConnections/WordPressExternalConnection.php

```diff
@@ -658,7 +658,15 @@ public function push( $post_id, $args = array() ) {
 			\Distributor\Subscriptions\create_subscription( $post_id, $body_array['id'], untrailingslashit( $this->base_url ), $signature );
 		}
 
-		return $body_array['id'];
+		$remote_post = array(
+			'id' => $body_array['id'],
+		);
+
+		if ( ! empty( $body_array['push-errors'] ) ) {
+			$remote_post['push-errors'] = $body_array['push-errors'];
+		}
+
+		return $remote_post;
```

Comment:
Couple things here.

1. Since we are changing from returning just an ID to an array, we should change the dockblock for this function
2. This isn't a code review comment but because we are changing what this function returns, we'll need to be very clear in our release notes, so any 3rd party integrations that might utilize this `push` function can update accordingly, if needed

      2a. This also means we return different things in the `push` method for Internal and External connections now. Might be a factor once we start work on cleaning up and standardizing how we push and pull between connection types